### DISCUSSION TITLE
New name for `Inpaint at full resolution`

### DISCUSSION THREAD

**Discussion by LEv145** [Bing-su/adetailer#71] [Ideas]
As for me, the name of such a parameter is quite incomprehensible, how about using this: `Use "only masked" for inpaint area instead of "whole picture"`

**Comment by Bing-su** [comment]
I think the current name, taken from ddetailer, is fine.

**Reply by LEv145** [reply]
It's not a bad name, but I've seen a lot of people get confused and think that `Inpaint at full resolution` = `Whole image` in `img2img Inpaint`
